<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор мероприятий</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Подробнее</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="dlt_btn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>60</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Фиксированная дата:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>*Периодичность:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>На какой рабочий день:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>*Название:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Изменить цвет..</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="11" column="1" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Добавить новое событие</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Мероприятия в этот день</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Цвет ярлыка:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Выбрать...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">background-color: #00ffff;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
